.github/workflows/build-docker-images-dispatch.yml

```diff
@@ -0,0 +1,90 @@
+---
+name: Trigger specific image build
+
+on:
+  workflow_dispatch:
+    inputs:
+      product:
+        required: true
+        description: Product to build
+        type: choice
+        options:
+        - auth
+        - recursor
+        - dnsdist
```

Comment:
I was pondering fixing the value to `dnsdist` here but it will make backporting changes later harder so I'm OK with keeping it as it is now.

Reply:
I also had the same dilemma. I decided to keep it following the "build-packages" workflows but I have no issue cleaning that up.

Another doubt I had was with the `docker.yml` workflow. I cherry-picked the changes but that workflow only makes sense in `master` as it is run only with `on_schedule` events.

Reply:
Yeah, I think it makes sense to keep it simple for now and see how it goes :)